apps/studio/src/features/editing-experience/components/TipTapComponent.tsx

```diff
@@ -106,7 +99,8 @@ function TipTapComponent({
     ],
     content: data,
     onUpdate({ editor }) {
-      handleChange(path, editor.getJSON().content)
+      // setEditorState()
+      console.log(path, editor.getJSON().content)
```

Comment:
nit: remove this please! also what was `handleChange` last time? previously, this component didn't know how to handle changes and relied on its consumer telling it how to. 

now, it seems like it makes that determination itself?

Reply:
This was the previous `handleChange` which was being passed! Placeholder code used as page state hasn't been setup properly yet